src/2018/__dev/unconference.md

```diff
@@ -0,0 +1,72 @@
+# DartConf Unconference
+
+Most traditional conference sessions are videotaped and uploaded to sites
+like YouTube, so the main benefit of traveling to and attending a conference
+is the chance to meet and interact with other attendees and speakers,
+and to participate in open discussions with people with whom you share
+interests.
+
+An unconference (AKA a “barcamp”) is a set of special conference sessions
+where the conference attendees come up with session ideas and the sessions
+are more like discussions than presentations. The idea is to provide a more
+interactive experience compared to just attending talks with slides.
+For many people, the unconference sessions are the highlight of a conference.
+
+We are having a short unconference at the end of DartConf ’18. We hope you
+participate, both by suggesting unconference sessions, and attending these
+sessions.
+
+## Creating a session
+First, you do not need to be an expert in a topic in order to create a
+session for it. In fact, some of the best unconference sessions are created
+by people who merely want to learn more about some new thing. If you know
+someone who does know about the subject you can invite them to come and
+participate in the session, but that is not required.
+
+Unconference sessions are interactive, with lively discussions and sharing.
+If you just want to give a talk or quickly demo a project you have built,
+you should probably give a lightning talk instead (unless the idea is to
+have an open discussion about your project). The creator of the session
+should learn as much as the other session attendees.
+
+Unconference sessions are also good places to develop ideas. I’ve even seen
+unconference sessions that result in new ideas being implemented by the
+participants. Be open to ideas from the attendees.
+
+During the session, it is a good idea to make sure that everyone has a
+chance to contribute. A session should not become dominated by a small
+number of people (to the exclusion of others). Look around to see if
+there are people who look like they have an idea to contribute, but are
+shy or uncomfortable speaking in public. Ask them if they want to share
+their ideas or questions.
+
+Expect people to come and go from your session. This is called the
+"law of two feet".
```

Comment:
Add link
["law of two feet"](https://en.wikipedia.org/wiki/Open_Space_Technology#Law_of_two_feet)